articles/industrial-iot/tutorial-configure-industrial-iot-components.md

```diff
@@ -48,25 +48,21 @@ output of deployment script or reset the password
 * IoT Hub → IoT Edge → \<DEVICE> → Set Modules → OpcPublisher (for standalone OPC Publisher operation only)
 
 
-## Configuration options
+## OPC Publisher Configuration options
```

Comment:
Many of the options are still missing, as per https://github.com/Azure/Industrial-IoT/blob/main/docs/modules/publisher-commandline.md. 

I believe we should move to the Environment Variable format as in the github doc rather than promoting further the legacy format options.